docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md

```diff
@@ -102,14 +102,12 @@ $ vi /etc/multipath.conf
 >Copy the following into /etc/multipath.conf
 
 ```
-defaults
-{
+defaults{
     polling_interval 10
     max_fds 8192
 }
 
-devices
-{
+devices{
```

Comment:
Please put "device {" also on the same line, the way it is for defaults and devices.